examples/step-50/doc/intro.dox

```diff
@@ -1,90 +1,128 @@
-defines this particular problem.
+For defining $\mathbb{E}$, it is important to note that, as we are using local smoothing
+to define the multigrid hierarchy (see the @ref mg_paper "multigrid paper" for a description of
+local smoothing), the refinement level of a cell corresponds to that cells multigrid
+level. Now, let $N_{\ell}$ be the number of cells on level $\ell$
+(both active and non-active cells) and $N_{\ell,p}$ be the subset owned by process
+$p$. Assuming that the workload for any one processor is proportional to the number
+of cells owned by that processor, the optimal workload per processor is given by
+@f{align*}
+W_{\text{opt}} = \sum_{\ell}\frac1{n_{p}}\sum_{p}N_{\ell,p}=\frac1{n_{p}}\sum_{\ell} N_{\ell}.
+@f}
+Next, assuming a synchronization of work on each level (i.e., on each level of a vcycle,
+work must be completed by all processors before moving on to the next level), the
+limiting effort on each level is given by
+@f{align*}
+W_\ell = \max_{p} N_{\ell,p},
+@f}
+and the total parallel complexity
+@f{align*}
+W = \sum_{\ell} W_\ell.
+@f}
+Then we define $\mathbb{E}$ as a ratio of the optimal partition to the parallel
+complexity of the current partition
+@f{align*}
+  \mathbb{E} = \frac{W_{\text{opt}}}{W}.
+@f}
+For the example distribution above, we have
+@f{align*}
+W_{\text{opt}}&=\frac{1}{n_p}\sum_{\ell} N_{\ell} = \frac{1}{3} \left(1+4+4\right)= 3 \qquad
+\\
+W &= \sum_\ell W_\ell = 1 + 2 + 3 = 6
+\\
+\mathbb{E} &= \frac{W_{\text{opt}}}{W} = \frac12.
+@f}
+The value MGTools::workload_imbalance()$= 1/\mathbb{E}$ then represents the factor increase
+in timings we expect for GMG methods (vmults, assembly, etc.) due to the imbalance of the
+mesh partition.
 
-The things that are new are all those parts that concern the
-multigrid. In particular, this includes the following members of the
-main class:
-- <code>LaplaceProblem::mg_dof_handler</code>
-- <code>LaplaceProblem::mg_sparsity</code>
-- <code>LaplaceProblem::mg_matrices</code>
-- <code>LaplaceProblem::mg_interface_matrices_up</code>
-- <code>LaplaceProblem::assemble_multigrid ()</code>
-- <code>LaplaceProblem::solve ()</code>
-Take a look at these functions.
+_CITE MG PAPER_ contains a full discussion of the partition efficiency model
+and the effect the imbalance has on the GMG vcycle timing. In summary, the value
+of $\mathbb{E}$ is highly dependent on the degree of local mesh refinement used and has
+an optimal value $\mathbb{E} \approx 1$ for globally refined meshes. Typically for adaptively
+refined meshes, the number of processors used to distribute a single mesh has a
+negative impact on $\mathbb{E}$ but only up to a leveling off point, where the imbalance
+remains relatively constant for an increasing number of processors, and further refinement
+has very little impact on $\mathbb{E}$. Finally, $1/\mathbb{E}$ was shown to give an
+accurate representation of the slowdown in parallel scaling expected for the timing of
+a vcycle.
```

Comment:
You might want to add that there is some potential for asynchronous execution of different processors on different levels with purely nearest neighbor MPI communication, but that this can only cover up a small part of the imbalance because the wait for the immediate neighbor is a pretty strong barrier nonetheless - I could construct cases where a bad efficiency mesh is well-balanced and could run 2x faster than the model predicts, but the statistical expectation is that the efficiency describes it well.

Reply:
I added an additional paragraph.